### PR TITLE
Example async rest webapp

### PR DESCRIPTION
**Pull request by janbartel**
This is an example webapp showing how to use async servlets for rest calls.

### CODE REVIEW

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+	<application>async-rest-demo</application>
```
Comment (ludoch) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: remove application entry (now taken from your gcloud default project)

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	<module>async-rest-demo</module>
+	<vm>true</vm>
+	<version>1</version>
```
Comment (ludoch) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: remove version

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	<public-root>/static</public-root>
```
Comment (ludoch) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: remove public-root, this is causing the images not being seen.

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+	<precompilation-enabled>false</precompilation-enabled>
+	<threadsafe>true</threadsafe>
+	<sessions-enabled>false</sessions-enabled>
+</appengine-web-app>
```
Comment (ludoch) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: You'll also need a Dockerfile next to index.html as this is (for now) a Java8 runtime, so a custom one (the default one is only Java7)
Dockerfile content for now: (will change soon)
```
FROM gcr.io/google_appengine/jetty9-compat:githubheadasync
ADD . /app
```

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml (hunk 1 above)
Comment (janbartel) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: done

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml (hunk 2 above)
Comment (janbartel) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: done

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml (hunk 3 above)
Comment (janbartel) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: done. App now works.

File: managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml (hunk 4 above)
Comment (janbartel) [inline managedvms/async-rest/src/main/webapp/WEB-INF/appengine-web.xml]: Checked in.

**Comment by gregw** [comment]
@ludoch pushed the working version

**Comment by ludoch** [comment]
LGTM

**Pull request merged**